hooks/post/30-upgrade.rb

```diff
@@ -15,7 +15,7 @@ def remove_legacy_mongo
   if `rpm -q mongodb --queryformat=%{version}`.start_with?('2.')
     logger.warn 'removing MongoDB 2.x packages, config and log files.'
     Kafo::Helpers.execute('yum remove -y mongodb-2* mongodb-server-2* > /dev/null 2>&1')
-    Kafo::Helpers.execute('rm -rf /etc/mongod.conf /var/log/mongodb')
+    Kafo::Helpers.execute('rm -rf /etc/mongod.conf')
```

Comment:
Looking at this, why do we use `-r` on a single file? That's just asking for errors. We could also use pure ruby with `File.unlink('/etc/mongod.conf') if File.exist?('/etc/mongod.conf')`

Reply:
Good catch, sorry for merging so fast.

Reply:
I will make a new PR to fix it